chore: add warning for invalid render function of createRawSnippet (#12535)
sveltejs/svelte: packages/svelte/messages/client-warnings/warnings.md

> ## bad_raw_snippet_render

Reviewer: we use 'invalid' in comparable messages elsewhere, this would be the only use of `bad_`

```suggestion
## invalid_raw_snippet_render
```

> > The `render` function of `createRawSnippet` is expected to return the HTML for a single element. Ensure the HTML generated from `render` is correct.

Reviewer: ```suggestion
> The `render` function passed to `createRawSnippet` should return HTML for a single element
```
